test/e2e/simple_test.go

```diff
@@ -26,19 +27,25 @@ func TestE2ESimple(t *testing.T) {
 		t.Skip("skipping e2e test")
 	}
 
-	if os.Getenv("E2E_VERSIONS") != "" {
-		versionsStr := os.Getenv("E2E_VERSIONS")
-		versions := ParseVersions(versionsStr)
-		if len(versions) > 0 {
-			latestVersion = versions.GetLatest().String()
+	if os.Getenv("E2E_LATEST_VERSION") != "" {
+		latestVersion = os.Getenv("E2E_LATEST_VERSION")
+		isSemVer, _ := ParseVersion(latestVersion)
+		switch {
+		case isSemVer:
+		case latestVersion == "latest":
+		case len(latestVersion) == 8:
+			// assume this is a git commit hash (we need to trim the last digit to match the docker image tag)
+			latestVersion = latestVersion[:7]
```

Comment:
[Optional] This part could be refactored into a function and reused elsewhere, as the same code block is repeated later in the `TestMajorUpgradeToV2` test.

Reply:
I'll consider doing it later